appengine/rails-cloudsql-postgres/Gemfile

```diff
@@ -61,9 +61,9 @@ end
 gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # [START gems]
-# Added at 2017-08-23 15:24:52 -0700 by franknatividad:
-gem "pg", "~> 0.21.0"
+# Added at 2021-04-20 by trambui09:
```

Comment:
Let's get rid of these "Added" comments. They're not useful as part of a user-facing sample.